### DISCUSSION TITLE
第3节课作业(6班)

### DISCUSSION THREAD

**Discussion by maxchiron** [InternLM/Tutorial#274] [作业]
第3节课作业(6班)

**Comment by JackieGLQ** [comment]
微信名：老高
基础作业：https://z01krqqa831.feishu.cn/docx/EMs2dSD0YoQpSExNKDHc3dY0n9g#Yxtqdj88GoeKMixyn6HcPIDGnxc

**Comment by zhangzefeng92** [comment]
微信名称：捂上眼睛看世界
基础作业：https://www.toutiao.com/article/7322093756717711891/
![图片](https://github.com/InternLM/tutorial/assets/58167242/eec9289f-d1d9-426b-88a2-a93287bc0d08)

**Comment by BIT-WuZhiMing** [comment]
基础作业：

复现课程知识库助手搭建过程 (截图)
LangChain相关环境配置
 
![image](https://github.com/InternLM/tutorial/assets/148635592/c2b8ed8e-62d1-416d-9708-48e5401fcc6e)

数据收集
 
![image](https://github.com/InternLM/tutorial/assets/148635592/16d5cb76-22c7-46f4-a3aa-d34fecac9fd3)

构建检索问答链
 
![image](https://github.com/InternLM/tutorial/assets/148635592/464f3bf6-78a2-497f-8733-d02712999fd0)

最终结果
![image](https://github.com/InternLM/tutorial/assets/148635592/15792696-fd2b-471d-88e2-6ad74124ef01)

**Comment by yokebtc** [comment]
yoke - [https://app.yinxiang.com/fx/764fd1f7-e47a-4812-8691-7c90d066e032](https://app.yinxiang.com/fx/764fd1f7-e47a-4812-8691-7c90d066e032)

**Comment by RalphGuo** [comment]
微信ID-guos
自定检索问答链
![3](https://github.com/InternLM/tutorial/assets/58155843/b0afa8a8-04a0-4a7e-84df-56a220ccac1c)
